src/locales/en/common.json

```
@@ -577,7 +577,8 @@
       "settings": "Settings",
       "addressBook": "Address book",
       "supportChat": "Support chat",
```

```suggestion
      "supportChat": "Support",
```

"Support chat" and "Support conversations" are too similar. Because "Support chat" just opens the main Instabug dialog and we now have a dedicated item for support conversations - we should rename this to just "Support".